server.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable */
```

Comment:
@marijohannessen do you have Prettier set up in your code editor? Letting Prettier format your code might allow you to remove the eslint-disable comment.